src/dumper_thread.c

```diff
@@ -230,7 +230,8 @@ load_rpm(const char *fullpath,
          int changelog_limit,
          struct stat *stat_buf,
          cr_HeaderReadingFlags hdrrflags,
-         GError **err)
+         GError **err,
```

Comment:
Please keep code consistent and put GError as a last argument, this is an internal function anyway, you don't have to worry about API/ABI breakage